packages/sdk/src/utils/gas.ts

```diff
@@ -0,0 +1,182 @@
+import {getNetworkByChainId, networks} from '@holographxyz/networks'
+import {Address, Block, PublicClient, Transaction} from 'viem'
+import {GasPricing} from './types'
+
+export function getTestGasLimit(chainId: number) {
+  let testGasLimit: bigint
+
+  switch (chainId) {
+    case networks.mantle.chain:
+    case networks.mantleTestnet.chain:
+      testGasLimit = BigInt(1000000000)
+      break
+    default:
+      testGasLimit = BigInt(10000000)
+      break
+  }
+
+  return testGasLimit
+}
+
+export function getGasEstimationAddress(chainId: number): Address {
+  // Optimism and zora uses a different address for the zero address so we use that instead to simulate the tx while estimating gas
+  // NOTE: The different address is the address of the Wrapped ETH contract
+  let destinationFrom: Address
+
+  switch (chainId) {
+    case networks.optimism.chain:
+    case networks.optimismTestnetGoerli.chain:
+    case networks.optimismTestnetSepolia.chain:
+    case networks.zora.chain:
+    case networks.zoraTestnetGoerli.chain:
+    case networks.zoraTestnetSepolia.chain:
+    case networks.base.chain:
+    case networks.baseTestnetGoerli.chain:
+    case networks.baseTestnetSepolia.chain:
+      destinationFrom = '0x4200000000000000000000000000000000000006'
+      break
+
+    case networks.arbitrumOne.chain:
+      destinationFrom = '0x82aF49447D8a07e3bd95BD0d56f35241523fBab1'
+      break
+
+    case networks.arbitrumTestnetGoerli.chain:
+      destinationFrom = '0xEe01c0CD76354C383B8c7B4e65EA88D00B06f36f'
+      break
+
+    case networks.arbitrumTestnetSepolia.chain:
+      destinationFrom = '0x0b39384cd4680A0287b03F1D6e60dD5cfAF4e473'
+      break
+
+    default:
+      destinationFrom = '0x0000000000000000000000000000000000000000'
+      break
+  }
+
+  return destinationFrom
+}
+
+// This function is here to accomodate instances where a network has a minimum BaseBlockFee
+export function adjustBaseBlockFee(chainId: number, baseBlockFee: bigint): bigint {
+  const network: string = getNetworkByChainId(chainId).key
```

Comment:
Can you add fuji to this case. And set it as 3 Gwei for binance and binance testnet

Reply:
isn't avalancheTestnet the same as fuji?

Reply:
@BravoNatalie no you are right. `avalancheTestnet` is fine. `fuji` is the short name version.